packages/orchestrator-ui-components/src/components/WfoSubscription/WfoSubscriptionGeneralSections/WfoSubscriptionFixedInputSection.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import { useTranslations } from 'next-intl';
+
+import { SubscriptionKeyValueBlock } from '@/components';
+import { SubscriptionDetail } from '@/types';
+
+interface WfoSubscriptionFixedInputSectionProps {
+    subscriptionDetail: SubscriptionDetail;
+}
+
+export const WfoSubscriptionFixedInputSection = ({
+    subscriptionDetail,
+}: WfoSubscriptionFixedInputSectionProps) => {
+    const t = useTranslations('subscriptions.detail');
+
+    const fixedInputBlockData = subscriptionDetail.fixedInputs.map(
```

Comment:
How about only passing the fixedInputs to the component?

Reply:
fixedInputs: FieldValue[];

Should I make it this type, because that is what we have for fixedInputs ?